### ISSUE TITLE
problem loading annotations on initial page load. 

### ISSUE THREAD

**Issue by ShannonRice** [Harvard-ATG/annotationsx#47]
This is a problem with the page loading annotations correctly.  

When I first go to "The Phoenix" assignment in the English 102H course, 
https://canvas.harvard.edu/courses/6571/external_tools/10610

The annotations from lines 1-13 do not display at all.  (see screenshot 1) 
(There are also other clusters of annotations not displaying, if you compare the two screenshots.... )

![annotationsx1](https://cloud.githubusercontent.com/assets/2422354/11183719/701b8904-8c41-11e5-9a40-5423f70e4579.png)

After I click the "Instructor" button, they all load and display properly, and if I then click "Public", they all load correctly.  
![annotationsx2](https://cloud.githubusercontent.com/assets/2422354/11183721/72060348-8c41-11e5-8929-e2f043f6f9e8.png)

For some reason they do not load initially.  

This is a problem, since students would expect to see the annotations. I'm not sure if it is specific to this assignment, or more wide spread.  I will continue testing to see if I can recreate on another site.

**Comment by arthurian** [comment]
Follow-up for posterity: we figured out this issue was caused by the pagination behavior of the annotation assignment. By default, the assignment loads annotations in batches of 20 and for some reason, the first batch that is loaded isn't the first batch of text annotations, so it appears as if that first batch is lost. The work around is to edit the assignment and increase the pagination limit to avoid the problem.

This issue is addressed in https://github.com/lduarte1991/hx-annotations-lti/pull/44, which increases the default pagination limit so that users won't encounter this issue in practice. We could configure the default to be something like 1000.
